public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -272,6 +272,7 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
   );
 
   const updateExpressionAndDatasource = useSetExpressionAndDataSource();
+  const isOptimizeReducerEnabled = config.featureToggles.alertingOptimizeReducerInUI ?? false;
```

Comment:
Super minor point - perhaps when we have feature toggles that are UI only, we could name them `alertingUI...`?

Reply:
good point, we should update the existing ones though...maybe from now? I can update this one

Reply:
done!